deploy/kubernetes/latest/controller.yaml

```diff
@@ -42,7 +42,7 @@ spec:
               mountPath: /csi
         - name: gce-pd-driver
           imagePullPolicy: Always
-          image: gcr.io/google-containers/volume-csi/gcp-compute-persistent-disk-csi-driver:v0.1.0.alpha
+          image: gcr.io/dyzz-csi-staging/csi/gce-pd-driver:latest
```

Comment:
Consider using kustomization for this purpose

Reply:
Instead of calling this folder "latest", I would maybe consider "dev"

Reply:
I think I'll make another branch called dev and leave the master branch with the v0.1.0.alpha version since I have a feeling a lot of people might pull master to get the driver up and running

Reply:
What I was thinking was having maybe two directories: stable and dev.  Stable on master will contain whatever is the latest stable version.

Reply:
that makes sense to me. saves branch switching as well as doesn't have an explosion of version numbers